### PR TITLE
Adds new component TextArea

### PR DESCRIPTION
**Pull request by tumentumurchudur**
This PR adds a new component `TextArea`.

## Props
- elementProps: object - allows the parent component to pass props to `textarea` such as onChange, onBlur, placeholder, etc.
- primaryColor: string - border color of the component when it has focus
- rows: number - determines the number of rows
- valid: bool - determines whether the value in the component is valid

screen shot 2016-09-14 at 9 51 03 am

### CODE REVIEW

**Comment by jtanner** [comment]
Once you take care of the conflict (demo/app.js was removed in #432) I'll give this a 🚢 .

**Comment by tumentumurchudur** [comment]
Added documentation. Ready for review.

screen shot 2016-09-22 at 4 04 01 pm

File: src/components/TextArea.js [hunk 1]
```diff
@@ -54,7 +54,9 @@ const TextArea = React.createClass({
       >
         <textarea
           {...elementProps}
-          ref='textarea'
+          ref={ref => {
+            this.textarea = ref;
+          }}
```
Comment (jtanner) [inline src/components/TextArea.js]: What's the advantage of doing it this way instead of `this.refs.textarea`?

**Comment by jtanner** [comment]
Besides my comment I think this is ready to :ship:

File: src/components/TextArea.js (hunk 1 above)
Comment (tumentumurchudur) [inline src/components/TextArea.js]: `this.refs.textarea` no longer works and throws a console error. It may have to do with the latest version of React.

**Comment by jtanner** [comment]
:shipit: ⚡

**Pull request merged**